src/main/java/com/stripe/model/ExternalAccountTypeAdapterFactory.java

```diff
@@ -17,7 +17,7 @@
       return null; // this class only serializes 'ExternalAccount' and its subtypes
     }
 
-    final String SOURCE_OBJECT_PROP = "object";
+    final String sourceObjectProp = "object";
```

Comment:
Just OOC, I guess these were recased becaused they're `final` but not `static`?

Reply:
Yes, the rules define constants as "static final fields whose contents are deeply immutable and whose methods have no detectable side effects": https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names.

cc @remi-stripe as this answer your question as well.

Reply:
Makes sense. Thanks!